Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35882&attempt_id=1e4ad2f8-7aac-4a6b-9bc1-6f5414fcdfe0)

<details><summary>Partial log (click to expand)</summary><p>

```
src/core/cpufeatures.c: In function 'getCPUFeatures':
src/core/cpufeatures.c:97:2: error: #error Do not know how to get CPU features on Linux.
 #error Do not know how to get CPU features on Linux.
  ^~~~~
src/core/cpufeatures.c:78:24: warning: unused variable 'hwcap' [-Wunused-variable]
     unsigned long long hwcap = getauxval(AT_HWCAP);
                        ^~~~~
make: *** [Makefile:1325: src/core/libvapoursynth_la-cpufeatures.lo] Error 1
builder for '/nix/store/66v0aw6v0alkd5szx3qy976b0wjlwmbd-vapoursynth-R43.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/66v0aw6v0alkd5szx3qy976b0wjlwmbd-vapoursynth-R43.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/vapoursynth/default.nix
+++ b/pkgs/development/libraries/vapoursynth/default.nix
@@ -1,8 +1,8 @@
 { stdenv, fetchFromGitHub, pkgconfig, autoreconfHook
 , zimg, libass, python3, libiconv
 , ApplicationServices, nasm
-, ocrSupport ?  false, tesseract
-, imwriSupport? true,  imagemagick7
+, ocrSupport ?  false, tesseract ? null
+, imwriSupport? true,  imagemagick7 ? null
 }:
 
 assert ocrSupport   -> tesseract != null;
@@ -12,13 +12,13 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "vapoursynth-${version}";
-  version = "R40";
+  version = "R43";
 
   src = fetchFromGitHub {
     owner  = "vapoursynth";
     repo   = "vapoursynth";
     rev    = version;
-    sha256 = "1ycc3fdhhryp7hap80z3qmh89br31kcswzp8ai3wlc07zfvcrfck";
+    sha256 = "01yzxggjxr6fz3wj81z6vgp9m4jqddyk73i22kz2x620cpdgb9j9";
   };
 
   nativeBuildInputs = [ pkgconfig autoreconfHook nasm ];
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
     description = "A video processing framework with the future in mind";
     homepage    = http://www.vapoursynth.com/;
     license     = licenses.lgpl21;
-    platforms   = platforms.unix;
+    platforms   = platforms.x86_64;
     maintainers = with maintainers; [ rnhmjoj ];
   };
 

